Calling/Utils/MetadataModels.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Calling
```

Comment:
Are these types needing to be named like this because this is how they are in the JSON response?

If we were to sum up all of the classes in this class would it be "MetadataModels" or would it be like CallRecordingResponseModels?